src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,147 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int ARRAY_BEGINNING = 0;
+    private static final int EMPTY_SIZE = 0;
+
+    private int size;
+    private T[] elementStore;
+
+    ArrayList() {
+        size = EMPTY_SIZE;
+        elementStore = (T[]) new Object[DEFAULT_CAPACITY];
+    }
```

Comment:
```suggestion

    private int size;
    private T[] elementStore;

    ArrayList() {
        elementStore = (T[]) new Object[DEFAULT_CAPACITY];
    }
```
it is always 0 when toy create it.